scripts/code.angularjs.org-firebase/functions/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "functions-firebase-code.angularjs.org",
   "description": "Cloud Functions to serve files from gcs to code.angularjs.org",
   "engines": {
-    "node": "8"
+    "node": "10"
   },
   "dependencies": {
     "@google-cloud/storage": "^4.7.0",
     "firebase-admin": "^8.10.0",
     "firebase-functions": "^3.6.0"
   },
   "private": true
-}
+}
```

Comment:
Let’s keep this as it was. How did it get removed here?

Reply:
Sure. OOC why do you prefer to have the newline?

Reply:
It makes Git diffs cleaner. If a file is not ended like that & you add something to the end of it, you modify the existing last line by adding a new line character to it & then have the new stuff. This modification is problematic & may generate more frequent conflicts. If all files end with a new line character, such cases just add new lines and don’t modify any.

Reply:
Cool. Good reasons!